include/cutlass/epilogue/collective/epilogue_tensor_broadcast.hpp

```diff
@@ -101,6 +102,9 @@ class EpilogueTensorBroadcast {
   static constexpr bool IsBinaryOp1Enabled = ThreadEpilogueOp::IsBinaryOp1Enabled;
   static constexpr bool IsUnaryOpEnabled = ThreadEpilogueOp::IsUnaryOpEnabled;
 
+  static constexpr bool PerColumnBias = PerColumnBias_;
+  using BiasStride = typename std::conditional<PerColumnBias, Stride<_0, _1, _0>, Stride<_1, _0, _0>>::type;
```

Comment:
We often prefer to use the `cute::` or `platform::` variants of things like `std::contditional` to work better with NVRTC.
```suggestion
  using BiasStride = cute::conditional_t<PerColumnBias, Stride<_0, _1, _0>, Stride<_1, _0, _0>>;
```